Extensions.Caching.PostgreSql/PostGreSqlCacheOptions.cs

```diff
@@ -45,11 +45,17 @@ public class PostgreSqlCacheOptions : IOptions<PostgreSqlCacheOptions>
 
         /// <summary>
         /// If set to true this instance of the cache will not remove expired items. 
-        /// Maybe on multiple instances scenario sharing the same database another instance will be responsable for remove expired itens.
+        /// Maybe on multiple instances scenario sharing the same database another instance will be responsable for remove expired items.
         /// Default value is false.
         /// </summary>
         public bool DisableRemoveExpired { get; set; } = false;
 
+        /// <summary>
+        /// If set to true no update of ExpiresAtTime will be performed when getting a cache item.
+        /// Default value is false.
+        /// </summary>
+        public bool DisableUpdateOnGetCacheItem { get; set; } = false;
```

Comment:
Instead of using the default as false and later negating it with (!) in database operations, why don't you turn it into something like `UpdateOnGetCacheItem` and leave the default as `true`